leshan-core/src/main/java/org/eclipse/leshan/core/request/execute/SingleArgument.java

```diff
@@ -0,0 +1,63 @@
+package org.eclipse.leshan.core.request.execute;
+
+public class SingleArgument {
+
+    private final int digit;
+
+    private final String value;
+
+    public SingleArgument(int digit, String value) {
+        this.digit = digit;
+        this.value = value;
+    }
+
+    public SingleArgument(int digit) {
```

Comment:
> This is only draft, it doesn't have proper format validation yet. 

Yep I guess this is here where you will put validation and raise a kind of  `InvalidSingleArgumentException`  / `InvalidArgumentException`